src/hotspot/share/gc/shenandoah/shenandoahMemoryPool.hpp

```diff
@@ -25,21 +26,46 @@
 #ifndef SHARE_GC_SHENANDOAH_SHENANDOAHMEMORYPOOL_HPP
 #define SHARE_GC_SHENANDOAH_SHENANDOAHMEMORYPOOL_HPP
 
-#ifndef SERIALGC
```

Comment:
Huh, I wonder what that ifndef was for. Apparently it was there since JEP189 appeared. Especially since SERIALGC is practically always defined - how did the includes even work? *scratches head*

Reply:
'Tis strange, to be sure.